Frontend/src/app/app-routing.module.ts

```diff
@@ -1,12 +1,16 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { LoginComponent } from './auth/pages/login/login.component';
-import { RegisterComponent } from './auth/pages/register/register.component';
-
+import { LogoutComponent } from './views/auth/logout/logout.component';
+import { LoginComponent } from './views/auth/login/login.component';
+import { RegisterComponent } from './views/auth/register/register.component';
+import { ProtectedPageComponent } from './views/test/protected-page/protected-page.component';
+import { AuthGuard } from './core/guards/auth.guard';
 
 const routes: Routes = [
   {
     path: 'auth',
+    canActivate: [ AuthGuard ],
```

Comment:
I can't access the login or register page with this here

Reply:
If you are logged in/authenticated, you won't be able to access the auth-related routes.